js/index.js

```diff
@@ -1 +1,145 @@
 // Your code goes here
+//Event 1: Mouseover
+let mouseAction = document.querySelectorAll('a');
+mouseAction.forEach(item => {
+    item.addEventListener('mouseover', (e) => {
+        item.style.border = "1px solid black";
+        item.style.borderRadius = "5px";
+        item.style.width = "70px"
+        item.style.textAlign = "center"
+            // preventDefault();
+
+
+    });
+})
+mouseAction.forEach(item => {
+        item.addEventListener('mouseleave', (e) => {
+            item.style.border = "";
+            item.style.borderRadius = "";
+            // preventDefault();
+
+
+        });
+    })
+    //Event 2 and 3: doubleClick single click plus append 
+var button = document.createElement('button'); // Create a <li> node
+var textnode = document.createTextNode(" Click for Fun"); // Create a text node
+button.appendChild(textnode); // Append the text to <li>
```

Comment:
Here, you could just have
```
const button = document.createElement("button");
button.innerText = "Click for Fun";
```

And it would work the same '' '